css/style.css

```diff
@@ -0,0 +1,303 @@
+    margin-right: 20px;
+    font-weight: bold;
+}
+.hero {
+    margin: 0 auto;
+    border: 2px solid blue;
+    max-width: 1150px;
+    display: flex;
+    justify-content: space-around;
+    position: relative;
+
+    color: white;
+}
+
+.hero__container {
+}
+
+.hero__screen {
+    width: 100%;
+    height: 500px;
+    background-position: center; /*centruje*/
+    background-repeat: no-repeat;
+    background-size: 400px;
+    align-items: center;
+    background-image: url(../images/screen.png);
+    transform: rotate(-8deg);
+    position: relative;
+    left: 200px;
+}
+.hero__header {
+    margin-top: 100px;
+    width: 250px;
+}
+.hero__paragraf {
+    width: 250px;
+}
+
+.hero__button {
+    min-width: 200px;
+    border: none;
+    color: white;
+    background-color: #1abc9c;
+    font-size: 1rem; /*ustalamy wielkość przycisku*/
+    padding: 0.5rem; /*ustalamy wielkość przycisku*/
+    cursor: pointer; /*dodajemy kursor po najechaniu na przycisk*/
+    transition: background-color 0.5s ease-in-out; /*ustawiamy przejście dla podświetlenia przycisku*/
+    border-radius: 8px;
+}
+
+.features {
+    background-color: black;
+    color: white;
+    border: 2px solid green;
+    width: 100%;
+}
+
+.features__header {
+    display: flex;
+    justify-content: center;
+}
+
+.features__header,
+.features__container {
+    max-width: 1150px;
+    margin: 0 auto;
+    border: 2px solid blue;
+}
+
+.features__container {
+    display: flex;
+    justify-content: center;
+}
+.features__container-list {
+    border: 2px solid yellow;
+    flex-basis: 33.333%;
+    text-align: center;
+}
+
+.features__header-list {
+}
+.features__item,
+.features__header-list {
+    text-align: center;
+}
+
+.fa-solid {
+    background-color: white;
+    color: black;
+    margin-top: 10px;
+
+    font-size: 2rem;
+}
+
+.features__container-span {
+    width: 50px;
+    height: 50px;
+    margin-top: 20px;
+    display: inline-block;
+    background-color: white;
+    border-radius: 50%;
+}
+
+.logo {
+    background-color: #ecf0f1;
+    padding: 15px;
+    border: 2px solid green;
+    display: flex;
+    justify-content: center;
+}
+.simple {
+    display: flex;
+
+    border: 2px solid green;
+}
+
+.simple__picture {
+    width: 100%;
+    height: 500px;
+    background-image: url(../images/imac.png);
+    background-position: center; /*centruje*/
+    background-repeat: no-repeat;
+    background-size: 400px;
+    align-items: center;
+    size: 300px;
+}
+.simple__text {
+    text-align: center;
+}
+
+.simple__picture,
+.simple__text {
+    border: 2px solid yellow;
+}
+
+.container__pricing {
+    background-color: #ecf0f1;
+    width: 100%;
+    text-align: center;
+    border: 2px solid green;
+}
+
+.pricing__header,
+.pricing__header-items {
+    max-width: 1150px;
+    margin: 0 auto;
+    border: 2px solid blue;
+}
+
+.pricing__header-items {
+    display: flex;
+}
+
+.pricing__item-first,
+.pricing__item-second,
+.pricing__item-third {
```

Comment:
Dlaczego 3 różne klasy? Te elementy wyglądają tak samo wiec spokojnie można użyć tej samej klasy.
Jedynie środkowy ma dodatkowe cechy więc użyłbym modyfikatora z BEM np. w  HTML wyglądałoby to tak: `class="pricing__item pricing__item--active"`